src/options/arith_options.toml

```diff
@@ -494,10 +494,11 @@ name   = "Arithmetic Theory"
 
 [[option]]
   name       = "nlCov"
-  category   = "expert"
+  category   = "regular"
   long       = "nl-cov"
   type       = "bool"
   default    = "true"
+  no_support = ["proofs"]
```

Comment:
Since this attribute is only allowed for regular options, we should guard this in mkoptions.py.

Reply:
I've added these checks (for legal no_support values, and to ensure it is a regular option).